app/src/format.py

```diff
@@ -46,6 +46,7 @@ def _get_bem_documents_to_show(
     # Ordered by the highest score of each chunk associated with the document
     documents: OrderedDict[Document, list[ChunkWithScore]] = OrderedDict()
     for chunk_with_score in chunks_with_scores[:chunks_shown_max_num]:
+        chunk_with_score.chunk = _add_ellipses(chunk_with_score.chunk)
```

Comment:
I think you should use a new variable to avoid unintentionally updating the DB record.